### PR TITLE
modify manual

### PR DESCRIPTION
**Pull request by boellner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,22 @@ This adapter serves as communication interface between Iobroker and your KNX IP
 The adapter allows to generate the iobroker communication objects automatically by importing an ETS group address xml export.  
 All generated communication objects are initially configured readable and writeable, values are fetched from the knx bus on adapter restart.
 
+# Installation
```
Comment (TA2k) [inline README.md:18]: The recommended way to install is via adapter list or click on the github icon in the adapter. 
We should not recommend manual installation this cause problems for the most of the users

Comment (boellner) [reply README.md:18]: can you add this?

**Comment by TA2k** [comment]
add installation part and short german instruction

**Review by boellner — COMMENTED** [review]
thx

**Pull request merged**